### PR TITLE
Added min_loss

### PR DESCRIPTION
**Pull request by marnunez**
From base

### CODE REVIEW

**Comment by aclouthier** [comment]
Have you seen that stopping criteria used previously? I've seen recommendations to base the stopping criteria on validation loss as opposed to training loss. I find the loss is pretty noisy, so setting a hard limit like that might stop the training a bit early.

**Comment by marnunez** [comment]
Truth be told, since I found your repository, I'm devouring a pytorch course :). So I haven't seen much of anything previously. I'll look into validation loss, see what I can modify
